src/components/DBT/DBTTaskList.tsx

```diff
@@ -333,6 +333,7 @@ export const DBTTaskList = ({
         title="Task"
         headers={['Command']}
         rows={rows}
+        height={75}
       />
```

Comment:
This is not needed , the DBTTaskList has the correct height after you added the height as a prop to the commmon `List` component and used it in only `Connections` page.

Reply:
Hey @Akshat-Jaiswal-8 thats okay, it can take the height of the content passed. 
Idea is when you dont define the height it should take up the content height of the row in the `List`.

The error you are solving for is only on `Connections` page, so ideally other things should remain the way they were.